js/index.js

```diff
@@ -1,491 +0,0 @@
-  randomPackage(packages);
-  organizePackage()}
-}, level); 
-
-
-function preload() {
-  // bgImg = loadImage('https://t3.ftcdn.net/jpg/00/88/98/18/360_F_88981880_YjJManMJ6hJmKr5CZteFJAkEzXIh8mxW.jpg');
-  bgImg = loadImage("img/background.png")
-  song = createAudio("sounds/music.mp3")
-  errorSound = createAudio("sounds/error.mp3")
-  gameOverSound = createAudio("sounds/game-over.mp3")
-  loadedSound = createAudio("sounds/loaded.mp3")
-}
-
-function setup() {
-    createCanvas(windowWidth-5, windowHeight-5);
-    button = createButton('Start Game');
-    button.position(width / 2 - button.width / 2 - 60 , height / 2 + 70);
-    button.mousePressed(startGame);
-    button.mouseOver(hover);
-    button.mouseOut(endHover);
-    button.style('font-size', '30px');
-    button.style('font-weight', 'bold')
-    button.style('color', color(213,0,41));
-    button.style('background-color', color(254,204,0));
-    button.style('border', 0);
-    button.style('padding', "10px");
-    button.style('box-shadow', "3px 4px #888888");
-
-
-    restartButton = createButton('Restart Game');
-    restartButton.position(width / 2 - restartButton.width / 2 , height / 2 + 40);
-    restartButton.mousePressed(restartGame);
-    restartButton.mouseOver(hover);
-    restartButton.mouseOut(endHover);
-    restartButton.style('font-size', '30px');
-    restartButton.style('font-weight', 'bold')
-    restartButton.style('color', color(213,0,41));
-    restartButton.style('background-color', color(254,204,0));
-    restartButton.style('border', 0);
-    restartButton.style('padding', "10px");
-    restartButton.style('box-shadow', "3px 4px #888888");
-    setInterval(moveBelt, 10);
-    
-  }
-  
-  function windowResized() {
-    resizeCanvas(windowWidth, windowHeight);
-  }
-
-
-
-  function draw() {
-    if (gameState === 0){
-      image(bgImg,0,0, width, height, 0, 0, bgImg.width, bgImg.height, COVER , LEFT)
-
-      noStroke()
-      fill(255,255,255,225)
-      rect(width / 2 - 325, height / 2 - 250, 650 , 400)
-
-    fill(213,0,41);
-    textAlign(CENTER);
-    textSize(32);
-    textStyle(BOLD)
-    text('Welcome to Delivery Man', width / 2, height / 2 - 190);
-
-    let line1 = "Type in the characters on packages to load them onto a truck.";
-    fill(50);
-    textAlign(CENTER);
-    textSize(20);
-    text(line1,  width / 2 - 300, height / 2 - 150 , 600, 400); 
-
-    let line2 = "Don't let the conveyor belt get too full, or it's game over!";
-
-    fill(50);
-    textAlign(CENTER);
-    textSize(20);
-    text(line2,  width / 2 - 300, height / 2 - 110 , 600, 400); 
-
-    let line3 = "Use UPPERCASE letters only";
-
-    fill(50);
-    textAlign(CENTER);
-    textSize(20);
-    text(line3,  width / 2 - 300, height / 2 - 70 ,600, 400); 
-
-    let line4 = "Are you ready to become the ultimate delivery man?";
-    fill(50);
-    textAlign(CENTER);
-    textSize(22);
-    text(line4,  width / 2 - 300, height / 2  , 600, 400); 
```

Comment:
Unnecessary to instantiate new variables every loop, just have the lines as constants outside the draw loop